src/storage/src/table/mod.rs

```diff
@@ -109,3 +110,42 @@ pub trait TableIter: Send {
         }
     }
 }
+
+/// Get vnode value with `indices` on the given `row`. Should not be used directly.
+fn compute_vnode(
+    row: &Row,
+    indices: &[usize],
+    vnodes: Arc<Bitmap>,
+    table_columns: &[ColumnDesc],
+    dist_key_indices: &[usize],
```

Comment:
Looks like these fields are for logging only. Since we've implemented the consistent hash correctly in most cases, I think it's okay to remove it now.